### ISSUE TITLE
Distorted Buttons in "Edit Target Definition" Dialog

### ISSUE THREAD

**Issue by jukzi** [eclipse-pde/eclipse.pde#727]
1. Goto Preferences "Target Platform"
2. click"Edit" Button
4. press a ALT key => All buttons but "Collapse All" disapear. NOK! they should be only disabled

![image](https://github.com/eclipse-pde/eclipse.pde/assets/51790620/b023e2fa-6bf2-4498-b869-56b65ce4a42b)

Same happens when clicking in the list ("Targlet Container Modular target") and then use Keyboard for navigation - for example "Cursor down"-key

**Comment by jukzi** [comment]
reproducable on 2022-12 (4.26) = no regression

**Comment by SudhindraKKulkarni** [comment]
@jukzi I analyzed this and I found out that this is a swt issue. This problem occurs when we are in the "light"theme, in "dark" theme it works fine. The problem is there since eclipse version 2022-03. So till 2021-12 it was working fine. The changes responsible for the problem is the because the api "wmColorChild" was overrriden in the "Button" class in order to fix the bugzilla bug 
https://bugs.eclipse.org/bugs/show_bug.cgi?id=577042
